File: wlxx-services/wlxx-system/wlxx-system-client/src/main/java/com/xzg/wlxx/system/client/feign/failback/TokenFallback.java
package com.xzg.wlxx.system.client.feign.failback;

import com.xzg.wlxx.system.client.entity.po.TokenPo;
import com.xzg.wlxx.system.client.feign.TokenProvider;
import org.springframework.stereotype.Component;

@Component
public class TokenFallback implements TokenProvider {

    @Override
    public TokenPo findToken(TokenPo tokenPo) {
        return null;
    }

    @Override
    public Boolean saveOrUpdate(TokenPo po) {
        return false;
    }
}

File: wlxx-services/wlxx-system/wlxx-system-client/src/main/kotlin/com/xzg/wlxx/system/client/feign/failback/TokenFallback.kt
package com.xzg.wlxx.system.client.feign.failback

import com.xzg.wlxx.system.client.entity.po.TokenPo
import com.xzg.wlxx.system.client.feign.TokenProvider

class TokenFallback : TokenProvider {
    override fun findToken(tokenPo: TokenPo?): TokenPo? {
        return null
    }

    override fun saveOrUpdate(po: TokenPo?): Boolean {
        return false
    }
}
